### ISSUE TITLE
error when starting metasploit

### ISSUE THREAD

**Issue by pretech86** [rapid7/metasploit-framework#11891]
## Steps to reproduce

How'd you do it?

1. ... install the latest version of metasploit 
2. ... download and run it dpkg -i ....
.

## Expected behavior

What should happen?

## start metasploit 

What happens instead?

Metasploit running on Kali Linux as root, using system database
A database appears to be already configured, skipping initialization
/opt/metasploit/ruby/lib/ruby/site_ruby/2.3.0/rubygems.rb:8:in `require': cannot load such file -- rbconfig (LoadError)
	from /opt/metasploit/ruby/lib/ruby/site_ruby/2.3.0/rubygems.rb:8:in `<top (required)>'
	from <internal:gem_prelude>:4:in `require'
	from <internal:gem_prelude>:4:in `<internal:gem_prelude>'

### Metasploit version

last version

### I installed Metasploit with:
- [ ] Kali package via apt
- [x] Omnibus installer (nightly)
- [ ] Commercial/Community installer (from http://www.rapid7.com/products/metasploit/download.jsp)
- [ ] Source install (please specify ruby version)

### OS

kali linux last version

**Comment by nixawk** [comment]
Please try the following method.

```
root@lab:/# apt install zlib1g-dev libpq-dev libpcap0.8-dev libsqlite3-dev
```

```
root@lab:/#  apt install metasploit-framework
```

```
root@lab:/usr/share#  mv metasploit-framework/ metasploit-framework2
root@lab:/usr/share#  git clone https://github.com/rapid7/metasploit-framework
```

```
root@lab:/usr/share#  cd metasploit-framework/
root@lab:/usr/share/metasploit-framework# bundle install
```

```
root@lab:~# msfconsole 
[-] ***rting the Metasploit Framework console...-
[-] * WARNING: No database support: could not connect to server: Connection refused
	Is the server running on host "localhost" (::1) and accepting
	TCP/IP connections on port 5432?
could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5432?

[-] ***
                                                  
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%     %%%         %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%  %%  %%%%%%%%   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%  %  %%%%%%%%   %%%%%%%%%%% https://metasploit.com %%%%%%%%%%%%%%%%%%%%%%%%
%%  %%  %%%%%%   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%  %%%%%%%%%   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%  %%%  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%    %%   %%%%%%%%%%%  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  %%%  %%%%%
%%%%  %%  %%  %      %%      %%    %%%%%      %    %%%%  %%   %%%%%%       %%
%%%%  %%  %%  %  %%% %%%%  %%%%  %%  %%%%  %%%%  %% %%  %% %%% %%  %%%  %%%%%
%%%%  %%%%%%  %%   %%%%%%   %%%%  %%%  %%%%  %%    %%  %%% %%% %%   %%  %%%%%
%%%%%%%%%%%% %%%%     %%%%%    %%  %%   %    %%  %%%%  %%%%   %%%   %%%     %
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  %%%%%%% %%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%          %%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


       =[ metasploit v5.0.25-dev-eb1592724a               ]
+ -- --=[ 1894 exploits - 1067 auxiliary - 329 post       ]
+ -- --=[ 547 payloads - 44 encoders - 10 nops            ]
+ -- --=[ 2 evasion                                       ]

msf5 >
```

- http://www.rapid7.com/products/metasploit/download.jsp
- https://github.com/rapid7/metasploit-framework/wiki/Setting-Up-a-Metasploit-Development-Environment

**Comment by busterb** [comment]
The omnibus nightly installer has not included Ruby 2.3 for some time. It seems like you have fairly old version as well.
